extensions/amp-viewer-integration/0.1/messaging.js

```diff
@@ -19,9 +19,16 @@ import {listen} from '../../../src/event-helper';
 import {dev} from '../../../src/log';
 
 const TAG = 'amp-viewer-messaging';
-const sentinel_ = '__AMPHTML__';
-const requestSentinel_ = sentinel_ + 'REQUEST';
-const responseSentinel_ = sentinel_ + 'RESPONSE';
+const APP = '__AMPHTML__';
+
+/**
```

Comment:
Trusted whitelist is in `viewer-impl.js` right now. It'd be good to move here, I agree. @chenshay Let's file an issue for this, but it's a lower priority for me at this time.